client/src/form/AddNewItemBar.js

```diff
@@ -28,15 +28,42 @@ export default function BasicTextFields(props) {
     e.preventDefault();
     console.log(productUrl);
     axios.post('/scraping', {
-        url: productUrl
+      url: productUrl
+    })
+    .then((response) => {
+      console.log(response);
+
+      axios.post("/lists/"+props.listId+"/products/new", {
+        url: productUrl,
+        name: response.data.title,
+        description: response.data.description,
+        image: response.data.image,
+        price: response.data.price
       })
-      .then(function (response) {
-        console.log(response);
-        //axios.post("/lists/:id/products/new", response.data: title, image, desc, price)
+      .then(function(response) {
+        console.log("will return nothing"+ response);
+        //can return product id or just re-render the product details from DB.
       })
-      .catch(function (error) {
+      .catch((error) => {
         console.log(error);
-      });
+      });  
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
I think it was decided yesterday that since we now have it working as a function, we will be calling the new product route instead